### ISSUE TITLE
Remove jQuery dependency

### ISSUE THREAD

**Issue by stevenvachon** [corysimmons/selectivizr2#4]
Many modern, modular apps are not using jQuery anymore.

**Comment by corysimmons** [comment]
Selectivizr requires some sort of selector engine and tbh I don't care enough about IE8 support to rewrite the project with a tiny selector engine.

Happy to test/accept a PR though. :)

**Comment by stevenvachon** [comment]
Me neither, haha.. I just figured that if it were small enough, I'd use it.

**Comment by corysimmons** [comment]
I should probably clarify that I'm not interested in IE8 enough to work on this project at all, but I'm happy to test out and merge PRs where the unmaintained version is not. I've commented on all the outstanding PRs over there, but I doubt anyone will bring them over here (it's been years). At some point I might try to bring them over, but for now Selectivizr2 does a good enough job.

TL;DR Too lazy to actually dev on Selectivizr, but I'll test/merge PRs and if I get super bored one night I'll migrate all those PRs over here. But yeah, it needs jQuery.
